tree_hugger/core/parser/php/queries.yml

```diff
@@ -1,4 +1,4 @@
-python_quaries:
+python_queries:
```

Comment:
Although most likely the top level key is going to go, but this key should have been `php_queries`

Reply:
I made the modification but forget to push, I can update if you want